lib/action_view/component/ar_monkey_patch.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class ActiveRecord::Base
```

Comment:
Were you able to take a look at this https://github.com/github/actionview-component/pull/129#discussion_r350170342 ?

Reply:
Will change it to ActiveModel.